pymodbus/pdu/pdu.py

```diff
@@ -102,12 +102,15 @@ class ModbusRequest(ModbusPDU):
 
     function_code = -1
 
-    def __init__(self, slave=0, **kwargs):  # pylint: disable=useless-parent-delegation
+    def __init__(self, slave=0, error_responses=True, truncated_responses=True, **kwargs):  # pylint: disable=useless-parent-delegation
```

Comment:
Error response have nothing to do with truncate, and that is something the app need to handle.

Reply:
> Error response have nothing to do with truncate

So I added that variable to allow tweaking the retry handler behavior for error responses similar to truncated responses since depending on desired behavior on may want to retry on error until successful.

> that is something the app need to handle

I had added this since I wasn't seeing a good approach once could use to handle this app side due to the retry handler and counter being part of the client.